src/editor/components/modals/revert-modal.jsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import PropTypes from "prop-types";
+import format from "date-fns/format";
+
+import Modal from "../../../shared/components/modal";
+import ModalTitle from "../../../shared/components/modal-title";
+import ModalContent from "../../../shared/components/modal-content";
+import ModalCall from "../../../shared/components/modal-call";
+import { TextButton } from "../../../shared/components/buttons";
+
+export default class RevertModal extends React.Component {
+  static propTypes = {
+    onCloseOrCancel: PropTypes.func.isRequired,
+    onRevert: PropTypes.func.isRequired,
+    visible: PropTypes.bool,
+    date: PropTypes.string
+  };
+
+  render() {
+    return (
+      <Modal
+        visible={this.props.visible}
+        onCloseOrCancel={this.props.onCloseOrCancel}
+        aboveOtherModals
+      >
+        <ModalTitle>Restore this revision?</ModalTitle>
+        <ModalContent>
+          {`This will restore your notebook to version from ${
```

Comment:
We should add more copy to this so the user has a better understanding of what this does. This functionality actually creates a new revision at the top of the revision stack, making it the latest revision. As a user, this is not clear from the text.

My suggestion:

```
This will restore your notebook to the version from ${dateLogicHere} by saving a copy of it as the most recent version. Your current changes will be preserved as a revision.
```